src/promptflow-tools/tests/test_open_source_llm.py

```diff
@@ -1,23 +1,33 @@
 import copy
 import os
 import pytest
+from typing import List, Dict
```

Comment:
@prakharg-msft could you please take a look at the failed tests in `tool_tests` CI? All oss llm tool tests failure and look similar "xx connection not found".
![image](https://github.com/microsoft/promptflow/assets/49483542/2d207caa-1a7d-46db-af32-b13e9d4cd557)

Reply:
Yes, I think this is related to how connection_Manager works in CI. I am seeking help from @brynn-code for this - https://github.com/microsoft/promptflow/pull/976#discussion_r1379303384

Reply:
this is fixed now.